external/fv3fit/tests/training/test_train.py

```diff
@@ -323,24 +328,6 @@ def test_dump_and_load_default_maintains_prediction(model_type):
     xr.testing.assert_equal(loaded_result, original_result)
 
 
-@pytest.mark.parametrize("model_type", ["sklearn_random_forest"])
-def test_train_predict_multiple_stacked_dims(model_type):
```

Comment:
Another test already handles predicting with multiple dimensions, and we don't support training in this mode any more.